draft/spec/index.html

```diff
@@ -172,47 +172,74 @@ <h2>Introduction</h2>
+          decision, implementers consider what is necessary to rebuild the objects from the files stored.
         </p>
+        <h3>Parsability</h3>
         <p>
-          By providing a specification for the file and directory layout on disk or in an object store, the OCFL is an
-          attempt at reducing, or even eliminating, the need for these transitions. As an application-independent
-          specification, conforming applications will natively "understand" the underlying file structure without
-          needing to first transition these contents to their own format.
-         </p>
+          One goal of the OCFL is to ensure objects remain fixed over time. This can be difficult as software and
+          infrastructure change and content is migrated. To combat this challenge, the OCFL ensures that both humans
+          and machines can understand the layout and corresponding inventory regardless of the software or
+          infrastructure used. This allows for humans to read the layout and corresponding inventory and understand it
+          without the use of machines. Additionally, if software were to become obsolete, the OCFL could easily be
+          understood by a light weight application, even without the full feature repository that might have been used
+          in the past.
+        </p>
+        <h3>Versioning</h3>
         <p>
-          While digital repository content changes relatively slowly, it is necessary to be able to track changes to
-          digital objects. Within the file and directory layout specification, the OCFL provides a simple structure to
-          efficiently capture versions of object contents so that all previous states of an object may be recovered
-          and examined.
+          Another need expressed by the community was the need to update and change objects, either the content itself
+          of the metadata associated with the object. The OCFL relies heavily on the prior art in the [[Moab]] Design
+          for Digital Object Versioning which utilizes forward deltas to track the history of the object. Utilizing
+          this schema allows implementers of the OCFL to easily recreate past versions of an OCFL object. Like with
+          determining what constitutes an OCFL object, the OCFL remains silent on when versioning should occur
+          recognizing again that this may differe from implementation to implementation.
+        </p>
+        <h3>Robustness</h3>
+        <p>
+          The OCFL also fills the need for robustness against errors, corruption, and migration. The versioning schema
+          ensures an OCFL object is robust enough to allow for the discovery of human errors. The fixity checking built
+          into OCFL via content addressable storage allows implementers to identify file corruption that might happen
+          outside of normal human interactions. Finally, the OCFL allows for methods that ease complications related to
```

Comment:
Can the "Finally.." sentence be made more specific? I'm not sure what this means

Reply:
also, should not use "Finally" because the next section starts with that

Reply:
@zimeon check to see if this change is better

Reply:
Yes except "The OCFL ~allows~ eases..."